### DISCUSSION TITLE
Export `PanelPositionType`

### DISCUSSION THREAD

**Discussion by lasficw** [bcakmakoglu/vue-flow#1420] [Ideas]
### Is there an existing issue for this?

- [X] I have searched the existing issues and this is a new bug.

### Current Behavior

As `PanelPosition` is deprecated we should use the `PanelPositionType`. This type is not exported as of now and can't get imported.

### Expected Behavior

The type `PanelPositionType` should be importable and the minimap and control packages should use the new type instead of the deprecated one.

### Steps To Reproduce

1. Import MiniMap or Control
2. Set position with `position` attribute
3. Editor states that `PanelPosition` is deprecated and that we should use `PanelPositionType` instead

**Comment by bcakmakoglu** [comment]
`PanelPositionType` is simply a string, there's nothing special to export or import here.

```vue
<Panel position="top-left">
 <!-- some content in here -->
</Panel>
```

Not sure what you would want to do with this type anyway? I can add an export to a future release for sure but this isn't a bug either.

**Comment by lasficw** [comment]
As stated, the minimap and the controls element use `PanelPosition` for positioning.

```
<MiniMap :position="PanelPosition.TopRight" pannable />
<Controls :position="PanelPosition.TopLeft" />
```

Those components don't support this new string based type and therefore we can only use the deprecated type for now. I worded it a bit weird, sorry about that!

**Reply by bcakmakoglu** [reply]
I see, will be fixed in the next release ^^

Relevant PR: https://github.com/bcakmakoglu/vue-flow/pull/1421

**Reply by bcakmakoglu** [reply]
Fix released with the latest patch please upgrade to the latest version of the core, minimap and controls package.
